gcp/eck/data-sources.tf

```diff
@@ -5,6 +5,15 @@ data "terraform_remote_state" "k8s" {
     path = "../gke/terraform.tfstate"
   }
 }
+
+locals {
+    master_roles = var.kibana_fleet_enabled ? "${var.master_pod_roles}, remote_cluster_client" : var.master_pod_roles
```

Comment:
@aag6z  wouldn't it be easier to just add remove cluster client to all roles instead of check if fleet is enable? Any downside to having remote_cluster_client on all roles?